ajax/libs/libphonenumber/package.json

```diff
@@ -22,7 +22,7 @@
         "files": [
           "asyoutypeformatter!(_test*)",
           "metadata!(fortesting*)",
-          "phone!(*_test*)"
+          "phone!(*test*)"
```

Comment:
btw, I think we can use !(*test*) in 3 rules, it's easy to read, and I tested it.